elements/pl-number-input/pl-number-input.py

```diff
@@ -233,20 +252,55 @@ def parse(element_html, data):
         data['submitted_answers'][name] = None
         return
 
-    # Convert to float or complex
-    try:
-        a_sub_parsed = pl.string_to_number(a_sub, allow_complex=allow_complex)
-        if a_sub_parsed is None:
-            raise ValueError('invalid submitted answer (wrong type)')
-        if not np.isfinite(a_sub_parsed):
-            raise ValueError('invalid submitted answer (not finite)')
-        data['submitted_answers'][name] = pl.to_json(a_sub_parsed)
-    except Exception:
-        if allow_complex:
-            data['format_errors'][name] = 'Invalid format. The submitted answer could not be interpreted as a double-precision floating-point or complex number.'
-        else:
-            data['format_errors'][name] = 'Invalid format. The submitted answer could not be interpreted as a double-precision floating-point number.'
+    if a_sub.strip() == '':
+        data['format_errors'][name] = get_format_string(allow_complex, allow_fractions, 'the submitted answer was blank.')
         data['submitted_answers'][name] = None
+        return
+
+    # support FANCY division characters
+    a_sub = a_sub.replace(u'\u2215', '/')  # unicode /
+    a_sub = a_sub.replace(u'\u00F7', '/')  # division symbol, because why not
+
+    if a_sub.count('/') == 1:
+        # Specially handle fractions.
+
+        if allow_fractions:
+            a_sub_splt = a_sub.split('/')
+            try:
+                a_parse_l = pl.string_to_number(a_sub_splt[0], allow_complex=allow_complex)
+                a_parse_r = pl.string_to_number(a_sub_splt[1], allow_complex=allow_complex)
+
+                if a_parse_l is None or not np.isfinite(a_parse_l):
+                    raise ValueError('invalid submitted answer')
```

Comment:
How about adding that is was with parsing numerator? As we can see what the issue is here. 

So, "Invalid numerator in submitted answer."